x-pack/plugins/security/public/authentication/login/components/login_form/login_form.scss

```diff
@@ -41,6 +41,14 @@
   + .secLoginCard {
     border-top: $euiBorderThin;
   }
+
+  &.secLoginCard-autoLogin {
```

Comment:
Will this be removed once the loading state is simplified as proposed by Larry?

If it remains for some other use, then we should tweak this name. The single `-` is intended for modifiers/states (e.g. `isLoading`). If this is simply a variation on the layout, then we could simply change to `__autoLogin`, for exmple.

Reply:
> Will this be removed once the loading state is simplified as proposed by Larry?

Yep, we don't need this anymore.

> If it remains for some other use, then we should tweak this name. The single - is intended for modifiers/states (e.g. isLoading). If this is simply a variation on the layout, then we could simply change to __autoLogin, for exmple.

Good to know! I was torn on this, in the code auto-login is treated as one of the loading states, but at the same time it's represented as a completely different layout too.